spec/system/create_proposal_spec.rb

```diff
@@ -34,13 +36,10 @@ def visit_component
     visit_component
   end
 
-  context "when category and scope are required" do
-    before do
-      allow(Decidim::SimpleProposal).to receive(:require_category).and_return(true)
-      allow(Decidim::SimpleProposal).to receive(:require_scope).and_return(true)
-    end
+  context "when category and scope are required," do
+    let(:settings) { { require_category: true, require_scope: true } }
 
-    context "without any scopes or categories" do
+    context "without any scopes or categories," do
```

Comment:
```suggestion
    context "without any scopes or categories" do
```

There is a typo here